java/java.hints/src/org/netbeans/modules/java/hints/errors/Utilities.java

```diff
@@ -3137,7 +3137,8 @@ public static boolean isCompatibleWithSwitchExpression(SwitchTree st) {
                             }
                             break;
                         } else {
-                            if (leftTreeName != null && leftTreeName.contentEquals(getLeftTreeName(statements.get(0)))) {
+			    Name exprTree = getLeftTreeName(statements.get(0));
```

Comment:
Please use not TABs but spaces.
Other changes as well.

Reply:
@junichi11 , done with the changes requested